pkg/sdk/logging/api/v1beta1/fluentbit_types.go

```diff
@@ -162,6 +162,8 @@ type InputTail struct {
 	PathKey string `json:"Path_Key,omitempty"`
 	// Set one or multiple shell patterns separated by commas to exclude files matching a certain criteria, e.g: exclude_path=*.gz,*.zip
 	ExcludePath string `json:"Exclude_Path,omitempty"`
+	// For new discovered files on start (without a database offset/position), read the content from the head of the file, not tail.
+	ReadFromHead string `json:"Read_From_Head,omitempty"`
```

Comment:
This might be better typed as a `bool` and formatted with `%t`, then you wouldn't have to worry about setting a default value for it.

Reply:
Good point, done.